parquet-encoding/src/main/java/org/apache/parquet/bytes/LittleEndianDataOutputStream.java

```diff
@@ -210,4 +212,12 @@ public final void writeDouble(double v) throws IOException {
     writeLong(Double.doubleToLongBits(v));
   }
 
+  public void close() {
+    try {
+      out.close();
+    }catch(IOException e){
+      throw new OutputStreamCloseException(e);
+    }
+  }
```

Comment:
make a helper method for this? closeUncaught(out) or something
alternatively the close method could throw an IOException